src/io/flutter/sdk/FlutterSettingsConfigurable.form

```diff
@@ -167,6 +167,15 @@
               <toolTipText value="Memory Profiling tab located inside of the Inspector view displays memory statistics."/>
             </properties>
           </component>
+          <component id="6d6f1" class="javax.swing.JCheckBox" binding="myTrackWidgetPerfCheckBox">
+            <constraints>
+              <grid row="5" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <text value="Enable Widget Rebuild and Repaint Profiler"/>
```

Comment:
We can alternately turn this on without an experimental flag. The UI is inactive unless the checkboxes in the perf tab are selected.

Reply:
removing this pref sounds good; we can adjust the verbiage in the Inspector UI itself to indicate that this is early access (`[ ] Track widget rebuilds (experimental)` or similar).

Reply:
done